src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -941,6 +942,259 @@ def log_structured_data(start_tag: str, dump_string):
         log_structured_data('==== json: ', json_str)
         log_structured_data('==== txt: ', txt_str)
 
+    @async_test_body
+    async def test_TC_DESC_2_1(self):
+        asserts.assert_true('PIXIT.DESC.AGGREVISION' in self.matter_test_config.global_test_params,
+                            "PIXIT.DESC.AGGREVISION must be included on the command line in "
+                            "the --int-arg flag as PIXIT.DESC.AGGREVISION:<device_type_revision>")
+
+        unique_ids = []
+        ROOT_NODE_DEVICE_TYPE = 0x0016
+        server_conformance = 0x001D
+        OnOff_deviceType = 0x0100
```

Comment:
the syntax of this feels off... constants are generally `UPPER_SNAKE_CASE`: https://peps.python.org/pep-0008/#constants